src/Juvix/Compiler/Backend/C/Translation/FromReg.hs

```diff
@@ -39,9 +40,42 @@ fromReg lims tab =
                 ListInitializer $
                   ExprInitializer (macroVar "BUILTIN_UIDS_INFO")
                     : map
-                      (\ci -> ListInitializer [ExprInitializer $ string (ci ^. Reg.constructorName)])
+                      ( \ci ->
+                          ListInitializer
+                            [ ExprInitializer $ string (ci ^. Reg.constructorName),
+                              ExprInitializer $ constrIsBinop (ci ^. Reg.constructorFixity),
+                              constrFixity (ci ^. Reg.constructorFixity)
+                            ]
+                      )
                       (Reg.userConstrs tab)
           }
+      where
+        constrIsBinop :: Maybe Fixity -> Expression
+        constrIsBinop = \case
+          Just fixity -> integer (fromEnum (isBinary fixity))
+          Nothing -> integer (0 :: Int)
+
+        constrFixity :: Maybe Fixity -> Initializer
+        constrFixity = \case
+          Just fixity ->
+            ListInitializer
+              [ ExprInitializer $ getPrec (fixity ^. fixityPrecedence),
+                ExprInitializer $ ExpressionVar $ getAssoc (fixity ^. fixityArity)
+              ]
+          Nothing -> ExprInitializer $ macroVar "APP_FIXITY"
+
+        getPrec :: Precedence -> Expression
+        getPrec = \case
+          PrecMinusOmega -> macroVar "PREC_MINUS_OMEGA"
```

Comment:
let's define these literal strings separately

Reply:
Oh dear! There are several dozen strings in this file, used only in the Reg-to-C translation, and it's very unlikely they will ever change or be used anywhere else because they correspond to the macro/variable names in the runtime. If they do change (because of a change in the runtime), you need to change the surrounding translation logic anyway, so it's never going to be a simple replacement.

I don't see the point of wasting time on introducing dozens of lines of useless code of the form 
```haskell
strPrecMinusOmega :: String
strPrecMinusOmega = "PREC_MINUS_OMEGA"
```
that's never going to help with anything. The only advantage of this I can see is that one can search for all uses of the definition. But one can equally well use Ctrl+F with "PREC_MINUS_OMEGA", so not much of an advantage.

This separate string-literal-definition is a rule-of-thumb that in this particular case becomes clearly counterproductive.

What's the point? Let's just merge this.